index.html

```diff
@@ -129,7 +129,335 @@ <h2 class="section-title">Schedule</h2>
       <!-- Begin "FAQ" -->
       <section id="faq" class="row">
```

Comment:
Sorry about the FAQ answers changing so frequently! I just got an updated version of the answers: https://docs.google.com/document/d/1ijSqPINjS7p-mywY918D7lCtdpMBjjxnSSLoyWUjIxU/edit#. Make sure that all links from the document are included in the HTML using `<a href="...">` tags